student-store-ui/package.json

```diff
@@ -10,7 +10,8 @@
     "axios": "^0.27.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.3.0"
+    "react-router-dom": "^6.13.0",
+    "react-router-hash-link": "^2.4.3"
```

Comment:
Since this isn't used anywhere, `react-router-hash-link` should be removed via `npm uninstall`